test/test_project.py

```diff
@@ -224,92 +495,110 @@ def test_export_survey_fields(self):
         for record in records:
             self.assertNotIn('redcap_survey_identifier', record)
             self.assertNotIn('demographics_timestamp', record)
-
+    
     @unittest.skipIf(skip_pd, "Couldn't import pandas")
+    @responses.activate
     def test_metadata_to_df(self):
         """Test metadata export --> DataFrame"""
+        self.add_normalproject_response()
         df = self.reg_proj.export_metadata(format='df')
         self.assertIsInstance(df, pd.DataFrame)
-
+    
     @unittest.skipIf(skip_pd, "Couldn't import pandas")
+    @responses.activate
     def test_export_to_df(self):
         """Test export --> DataFrame"""
+        self.add_normalproject_response()
+        self.add_long_project_response()
         df = self.reg_proj.export_records(format='df')
         self.assertIsInstance(df, pd.DataFrame)
         # Test it's a normal index
         self.assertTrue(hasattr(df.index, 'name'))
         # Test for a MultiIndex on longitudinal df
         long_df = self.long_proj.export_records(format='df', event_name='raw')
         self.assertTrue(hasattr(long_df.index, 'names'))
-
+    
     @unittest.skipIf(skip_pd, "Couldn't import pandas")
+    @responses.activate
     def test_export_df_kwargs(self):
         """Test passing kwargs to export DataFrame construction"""
+        self.add_normalproject_response()
         df = self.reg_proj.export_records(format='df',
             df_kwargs={'index_col': 'first_name'})
         self.assertEqual(df.index.name, 'first_name')
         self.assertTrue('study_id' in df)
-
+    
     @unittest.skipIf(skip_pd, "Couldn't import pandas")
+    @responses.activate
     def test_metadata_df_kwargs(self):
         """Test passing kwargs to metadata DataFrame construction"""
+        self.add_normalproject_response()
         df = self.reg_proj.export_metadata(format='df',
             df_kwargs={'index_col': 'field_label'})
         self.assertEqual(df.index.name, 'field_label')
         self.assertTrue('field_name' in df)
-
+    
     @unittest.skipIf(skip_pd, "Couldn't import pandas")
+    @responses.activate
     def test_import_dataframe(self):
         """Test importing a pandas.DataFrame"""
+        self.add_normalproject_response()
+        self.add_long_project_response()
         df = self.reg_proj.export_records(format='df')
         # grrr coerce implicilty converted floats to str(int())
-        for col in ['matrix1', 'matrix2', 'matrix3', 'sex']:
-            df[col] = map(lambda x: str(int(x)) if pd.notnull(x) else '', df[col])
+        # for col in ['matrix1', 'matrix2', 'matrix3', 'sex']:
```

Comment:
It's better just to remove old code than comment it out, that's why we have version control :)